store/store.go

```diff
@@ -6,10 +6,9 @@ import (
 	"errors"
 	"sync"
 
-	"go.uber.org/multierr"
-
 	"github.com/celestiaorg/optimint/state"
 	"github.com/celestiaorg/optimint/types"
+	"go.uber.org/multierr"
```

Comment:
Our convention is to split imports into few groups:
 1. standard libraries
 1. external deps
 1. internal deps